src/screens/StartHardwareWallet.js

```diff
@@ -128,9 +129,6 @@ class StartHardwareWallet extends React.Component {
         xpub,
         hardware: true,
       });
-      LOCAL_STORE.markBackupDone();
-
-      this.props.history.push('/wallet/');
```

Comment:
After starting the wallet we must change to the wallet screen. How is it done now? Through the sagas?

Reply:
This was here before to avoid changing the screen too fast (i.e. flashing the scren) before showing the loading addresses screen.
This has been fixed and is no longer needed, the saga will redirect to the loading addresses screen when required.